lib/ntp.js

```diff
@@ -145,7 +145,7 @@ export class Ntp {
 
 function getLogger () {
   if (Meteor.isServer) {
-    return Npm.require('debug')('kadira:ntp');
+    return require('debug')('kadira:ntp');
```

Comment:
If `Npm.require` is working again, we should switch back to it - it prevents `debug` from being bundled in the client, unlike using `require`.